src/types.ts

```diff
@@ -24,12 +24,16 @@ export interface AgentOSInfo extends AgentVersionInfo {}
  * @extends eg.agent.AgentVersionInfo
  * @typedef - Browser information <ko>브라우저 정보</ko>
  * @property - Whether the browser is webkit <ko>브라우저가 webkit인지 여부</ko>
+ * @property - Version of the webkit. If it is not webkit, "-1" is returned.<ko>webkit의 버전. webkit가 아닌 경우 "-1"을 반환한다.</ko>
```

Comment:
How about using `null` instead?
To make users can check whether version exists easily.
```ts
if (!info.webkitVersion) return;
```

Reply:
@WoodNeck 

You can see whether it is with `info.webkit`.

Here, if there is no version, all "-1" is returned.

Reply:
I think it's okay then 👍